.github/workflows/checks.yml

```diff
@@ -62,6 +62,7 @@ jobs:
         run: |
           cd ./docs
           pip install mkdocs-material
+          pip install mkdocs-redirects
```

Comment:
pls update `on-push-to-main.yml` as well

Reply:
Haven't noticed, thank you